### ISSUE TITLE
[EN][Quest][Desolace] Delicate Negotiations

### ISSUE THREAD

**Issue by Rhekan** [Atlantiss/BugTracker#3573]
[NEVERMIND - WORKED WHEN I TRIED AGAIN A FEW HOURS LATER]
http://www.wowhead.com/quest=14318/delicate-negotiations
After you kill a few Magram Defenders, this appears in the chat: "Khan Kammah yells: Enough! What is this about?!". However, all NPC's disappear and the NPC necessary to complete the quest (Khan Kammah - http://www.wowhead.com/npc=36163/khan-kammah ) doesn't spawn.

**Comment by Aival94** [comment]
did you mount npc for the quest as for me it worked before.

**Comment by Rhekan** [comment]
yup, I did, and he took me to the Fatherstone (though there was no icon for the spear in the action bar, but arriving at the Fatherstone triggered the next stage of the quest anyway). Then he disappeared and several friendly and hostile NPCs spawned and started fighting (just like they're supposed to do). But the NPC required to complete the quest didn't appear for me and I couldn't turn it in.

**Comment by Rhekan** [comment]
nevermind, I had tried it twice before and it didn't work (I even relogged between the attempts), but now, a few hours later, it worked and the NPC appeared

**Comment by Deonorth** [comment]
witam,
zalicza quest ale nie pojawia się np, któremu trzeba go oddać.. z tego co wiem to u różnych osób działa a u części nie. Quest jest ważny bo blokuje chaina niezbędnego do achievementa w Desolace

**Comment by Deonorth** [comment]
bump

**Comment by Deonorth** [comment]
dalej nie działa

**Comment by Larousene** [comment]
Quest is not working properly, NPC hasn't appear and i can't turn quest

**Comment by Deonorth** [comment]
bump, still not working

**Comment by Aival94** [comment]
Quest is working but just need to be close to the place of the flag to complete it when you kill the mobs.

The npc is moving you normally there but is moving kind of slower and is missing the spell to put the flag there.

Should work like that: https://youtu.be/wFpMQM6lU1Q

**Comment by Deonorth** [comment]
i know that but still don't work.. i finished this quest but i can't  back to the NPC..
